sycl/doc/extensions/ITTAnnotations/ITTAnnotations.rst

```diff
@@ -0,0 +1,79 @@
+ITT annotations support
+=======================
+
+This extension enables a set of functions implementing
+the Instrumentation and Tracing Technology (ITT) functionality
+in SYCL device code.
+
+There are three sets of functions defined by this extension,
+and they serve different purposes.
+
+User APIs
+---------
+
+The user code calling these functions must include the corresponding header
+file(s) provided by `ittnotify` project (TBD: reference ITT repo here).
```

Comment:
Since this is ReST and not Markdown, you need to replace each single backquote by 2 single-backquote everywhere to have code-like formatting...

Reply:
Thank you!